app/controllers/application_controller.rb

```diff
@@ -1,6 +1,7 @@
 class ApplicationController < ActionController::Base
   helper_method :current_user
   helper_method :logged_in?
+  helper_method :creator_user
```

Comment:
@idprietop what the **creator_user** method does. As a tip: be sure to name variables and methods with good English they need to be readable to other devs.

Reply:
The creator_user method receives the user_id that relates an article to a user and returns the user that contains that id. Thanks for the tip I will make the change and I will keep it in mind for next methods.